website/static/js/code-blocks-buttons.js

```diff
@@ -31,6 +31,7 @@ window.addEventListener('load', function() {
   );
 
   const clipboard = new ClipboardJS('.btnClipboard', {
+    /* eslint-disable object-shorthand */
```

Comment:
Not gonna use the shorthand as this is sent down to browsers without transpiling.